dftimewolf/lib/exporters/local_filesystem.py

```diff
@@ -44,8 +50,18 @@ def SetUp(self, target_directory=None):  # pylint: disable=arguments-differ
   def Process(self):
     """Checks whether the paths exists and updates the state accordingly."""
     for _, path in self.state.input:
-      self._CopyFileOrDirectory(path, self._target_directory)
-      print('{0:s} -> {1:s}'.format(path, self._target_directory))
+      if not self._compress:
+        full_paths = self._CopyFileOrDirectory(path, self._target_directory)
+        print('{0:s} -> {1:s}'.format(path, self._target_directory))
+        for path_ in full_paths:
+          self.state.StoreContainer(containers.FSPath(path=path_))
+      else:
+        try:
+          tar_file = utils.Compress(path, self._target_directory)
+          self.state.StoreContainer(containers.FSPath(path=tar_file))
```

Comment:
For consistency, could you add a print statement like above to indicate where the files are going?

Reply:
Done.